### PR TITLE
Fix table header scrolling #LMR-1742

### PR DESCRIPTION
**Pull request by livthomas**

### CODE REVIEW

File: src/app/view/perspectives/table/header/column-group/single-column/input/table-column-input.component.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        event.preventDefault();
+        event.stopPropagation();
+        return;
+      case KeyCode.ArrowUp:
+      case KeyCode.ArrowDown:
+        event.preventDefault();
+        return;
+    }
+
+    if (DISABLED_CHARACTERS.includes(event.key)) {
```
Comment (kubedo8) [inline src/app/view/perspectives/table/header/column-group/single-column/input/table-column-input.component.ts]: What about case when user paste text with disabled characters here? Is this covered?

File: src/app/view/perspectives/table/header/table-header.component.ts [hunk 2]
```diff
@@ -57,7 +57,11 @@ export class TableHeaderComponent implements OnInit, OnChanges {
   public singleCollection$: Observable<boolean>;
   public cursor: TableHeaderCursor;
 
-  public constructor(private renderer: Renderer2, private store$: Store<AppState>) {}
+  public constructor(
+    private element: ElementRef<HTMLElement>,
+    private renderer: Renderer2,
```
Comment (kubedo8) [inline src/app/view/perspectives/table/header/table-header.component.ts]: Are these properties needed?

File: src/app/view/perspectives/table/header/column-group/single-column/input/table-column-input.component.ts (hunk 1 above)
Comment (livthomas) [reply src/app/view/perspectives/table/header/column-group/single-column/input/table-column-input.component.ts]: Good point!

File: src/app/view/perspectives/table/header/table-header.component.ts (hunk 2 above)
Comment (livthomas) [reply src/app/view/perspectives/table/header/table-header.component.ts]: Not anymore.

**Pull request merged**